compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/util/Constants.java

```diff
@@ -39,6 +39,7 @@ private Constants() {
     public static final String OPEN_SEALED_ARRAY = "*";
 
     public static final int INIT_METHOD_SPLIT_SIZE = 50;
+    public static final int LISTENER_COUNT_PER_METHOD = 50;
```

Comment:
Is 50 working. I thought it should be 25?

Reply:
I checked again and 50 was working, even with test coverage. When running with no test coverage, method too large came with 220 listeners. When I was showing http tests running with coverage during the meeting, I think there I had split by 100 listeners and that's why that method too large stack trace showed.

Reply:
Since 25 is better anyway, changed it to 25.